src/styles/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: 100%;
+  padding: 32px 16px 16px;
+  border: 1px solid $color5;
+  border-radius: 5px;
```

Comment:
This border-radius value is used three times. I suggest putting it in a variable, naming it something like `$radius-small`.